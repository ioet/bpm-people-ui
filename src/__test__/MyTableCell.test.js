import React from 'react';
import BpmTableCell from '../component/presentational/BpmTableCell';

describe('<BpmTableCell />', () => {
  test('Component Renders correctly', () => {
    const appWrapper = shallow(<BpmTableCell />);

    expect(toJson(appWrapper))
      .toMatchSnapshot();
  });
});
