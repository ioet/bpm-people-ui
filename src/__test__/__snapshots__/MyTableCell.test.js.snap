// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BpmTableCell /> Component Renders correctly 1`] = `
<BpmTableCell
  classes={
    Object {
      "cell": "BpmTableCell-cell-1",
    }
  }
/>
`;
